src/libs/ReportUtils.ts

```diff
@@ -3943,7 +3943,7 @@ function canEditFieldOfMoneyRequest(reportAction: OnyxInputOrEntry<ReportAction>
     }
 
     if (fieldToEdit === CONST.EDIT_REQUEST_FIELD.REPORT) {
-        return getOutstandingReportsForUser(moneyRequestReport?.policyID, moneyRequestReport?.ownerAccountID, allReports ?? {}).length > 0;
+        return getOutstandingReportsForUser(moneyRequestReport?.policyID, moneyRequestReport?.ownerAccountID, allReports ?? {}).length > 1;
```

Comment:
I think this is ok for now since we'll CP this PR, but ultimately we also want to handle the case where the user has 1 available report and the available reportID is not the same as the transaction.reportID. Basically this flow:

1. Track an expense
2. Create a workspace
3. Create a report
4. Go to the tracked expense
5. Notice that you can see the `Report` field, but not edit it even though the user has an available report